daemons/dss-checkout-sfn/app.py

```diff
@@ -23,12 +24,29 @@
 dss.Config.set_config(dss.BucketConfig.NORMAL)
 email_sender = dss.Config.get_notification_email()
 default_checkout_bucket = dss.Config.get_s3_checkout_bucket()
-
 replica = "aws"
+DEBUG = False
 
 for client_name, client_class in chained_lambda_clients():
     chainedawslambda.aws.add_client(client_name, client_class)
 
+logger = logging.getLogger()
+if DEBUG:
+    logger.setLevel(logging.DEBUG)
+else:
+    logger.setLevel(logging.INFO)
```

Comment:
Please use the dss logger, and don't set levels here. Setting of log levels should not be done in library code.

Reply:
removed debugging code